pyrdp/player/MainWindow.py

```diff
@@ -32,13 +34,14 @@ def __init__(self, bind_address: str, port: int, filesToRead: [str]):
             "closeTabOnCtrlW": True     # Allow user to toggle Ctrl+W passthrough.
         }
 
-        self.liveWindow = LiveWindow(bind_address, port, self.updateCountSignal, self.options)
-        self.replayWindow = ReplayWindow(self.options)
+        self.liveWindow = LiveWindow(bind_address, port, self.updateCountSignal, self.options, parent=self)
+        self.replayWindow = ReplayWindow(self.options, mainWindow=self, parent=self)
```

Comment:
So I feel like I'm commenting this for the third time: You realize you are passing `self` twice for 2 different parameters, right? The whole `mainWindow` parameter is redundant because MainWindow **is** the parent of the ReplayWindow.

Reply:
god damn I keep thinking i removed everything from the previous implementation. will remove